shortcircuit-api/src/main/java/com/powsybl/shortcircuit/ShortCircuitParameters.java

```diff
@@ -82,7 +84,8 @@ public static ShortCircuitParameters load(PlatformConfig platformConfig) {
                         .setWithVSCConverterStations(config.getBooleanProperty("with-vsc-converter-stations", DEFAULT_WITH_VSC_CONVERTER_STATIONS))
                         .setWithNeutralPosition(config.getBooleanProperty("with-neutral-position", DEFAULT_WITH_NEUTRAL_POSITION))
                         .setInitialVoltageProfileMode(config.getEnumProperty("initial-voltage-profile-mode", InitialVoltageProfileMode.class, DEFAULT_INITIAL_VOLTAGE_PROFILE_MODE))
-                        .setVoltageRanges(getVoltageRangesFromConfig(config)));
+                        .setVoltageRanges(getVoltageRangesFromConfig(config))
+                        .setDetailedReport(config.getBooleanProperty("detailed-report", DEFAULT_DETAILED_REPORT)));
```

Comment:
This parameter can be tested in adding:

- in `src/test/resources/config.yml`:
```yml
detailed-report: false
```
- in `ShortCircuitParametersTest.testConfigLoader()`:
```java
assertFalse(parameters.isDetailedReport());
```

Reply:
You are right, I will add that right now, thank you